dist/ps-web-apis.d.ts

```diff
@@ -4,7 +4,6 @@ export interface WhoamiUserInfo {
     user_id: string;
     first_name?: string;
     last_name?: string;
```

Comment:
are we not referring here : https://github.com/spring-media/ps-whoami-web/blob/master/src/userinfo.ts#L3

Reply:
yes whoami-web (ps-web-apis version) has to updated afterwards as well. But it has no negative impact